setup.py

```diff
@@ -0,0 +1,10 @@
+from setuptools import setup
+
+setup(
+    name='frontend',
+    packages=['frontend'],
```

Comment:
setup(
    name='5yncr Tracker',
    version='0.0.1',
    packages=['syncr_tracker', ],
    license='AGPLv3',
)

Your's should match naming conventions of backend and tracker